modin/pandas/test/utils.py

```diff
@@ -466,6 +466,27 @@ def df_categories_equals(df1, df2):
         )
 
 
+def assert_empty_frame_equal(df1, df2):
+    """
+    Test if df1 and df2 are empty.
+
+    Parameters
+    ----------
+    df1: pandas.DataFrame or pandas.Series
+    df2: pandas.DataFrame or pandas.Series
```

Comment:
```suggestion
    df1 : pandas.DataFrame or pandas.Series
    df2 : pandas.DataFrame or pandas.Series
```
@anmyachev , don't we check this indent with doc-checker?

Reply:
We don't do doc check for this file.